### PR TITLE
this is ritika's firstc ommit

### PR DESCRIPTION
**Pull request by ritikabalouria**
dfyuvhioafnvogtroghj

### CODE REVIEW

**Comment by ritikabalouria** [comment]
kartik please review my changes , i will add the translatiosn later

**Comment by okanwaro** [comment]
Riitka this is awesome . Thnakd for your awesome Work!

File: trhird file.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bnmlpnn;pnpnp 9
```
Comment (okanwaro) [inline trhird file.txt]: PLease dfix this file!

Comment (ritikabalouria) [reply trhird file.txt]: fixed!

**Review by okanwaro — APPROVED** [review]

**Pull request merged**